invenio_pages/views.py

```diff
@@ -37,6 +38,16 @@
 )
 
 
+@blueprint.before_app_first_request
```

Comment:
For the moment this is the best option, trying to execute this code in the setup method fails as we need to fetch in the DB the pages and it seems that it is not initialized yet.